rpm/btrfs-balancer.spec

```diff
@@ -1,14 +1,17 @@
 Name:		btrfs-balancer
 Summary:	Automatic balancing service for btrfs filesystem
-Version:	1.1.0
+Version:	1.2.0
 Release:	1
 Group:		System/Filesystems
 License:	BSD
-URL:		https://bitbucket.org/jolla/tools-atruncate
+URL:		https://github.com/sailfishos/btrfs-balancer
 Source0:	%{name}-%{version}.tar.bz2
 
+BuildRequires:  pkgconfig(Qt5Core)
+BuildRequires:  pkgconfig(Qt5DBus)
+BuildRequires:  pkgconfig(keepalive)
+BuildRequires:  ssu-devel >= 0.37.9
 Requires:	systemd
-Requires:	libkeepalive-glib-tools >= 1.3.3
 
```

Comment:
As this accesses files in `/run/state/namespaces/Battery/`, should we also have a dependency on statefs?